packages/main/src/components/AnalyticalTable/scrollbars/VerticalScrollbar.tsx

```diff
@@ -52,10 +47,7 @@ export const VerticalScrollbar = forwardRef((props: VerticalScrollbarProps, ref:
   const classes = useStyles();
   const hasHorizontalScrollbar = tableRef?.current?.offsetWidth !== tableRef?.current?.scrollWidth;
 
-  const horizontalScrollbarSectionStyles = clsx(
-    classes.bottomSection,
-    hasHorizontalScrollbar && classes.bottomSectionWithScrollbar
-  );
+  const horizontalScrollbarSectionStyles = clsx(hasHorizontalScrollbar && classes.bottomSection);
 
   return (
```

Comment:
I couldn't mark the correct line, as there are no changes, but could you set the `width: '1px'` in line 77 to `'0.1px'`. This will prevent the border from looking bold. (especially with dark themes)

<img width="148" alt="image" src="https://user-images.githubusercontent.com/9749730/192739164-e9ed05e8-c5f4-4f68-8b2a-c0117b4596ee.png">